packages/styles/scss/components/filter-panel/_filter-panel.scss

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright IBM Corp. 2016, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import '../../globals/vars';
+@import '../../globals/imports';
+@import 'carbon-components/scss/components/accordion/keyframes';
+
+@mixin filter-panel {
+  .#{$prefix}--filter_heading {
+    padding: $spacing-05;
+  }
+  .#{$prefix}--filter_footer {
+    padding-bottom: $spacing-09;
+  }
+  .#{$prefix}--filter-panel__section {
+    background-color: $ui-01;
+    padding-bottom: calc((#{$spacing-12} + #{($spacing-05)}));
+    margin-top: $spacing-05;
+    overflow: initial;
+
+    @include carbon--breakpoint('sm') {
+      height: 100%;
+    }
+
+    @include carbon--breakpoint('md') {
+      height: 80%;
+      width: 60%;
+    }
+
+    @include carbon--breakpoint('lg') {
+      height: auto;
+      width: auto;
+      padding-bottom: $spacing-09;
+    }
+  }
+  .#{$prefix}--modal-body {
+    overflow-y: scroll;
+  }
+  .#{$prefix}--input_container {
+    width: 100%;
+  }
+  .#{$prefix}--input_container:focus {
+    outline: $spacing-01 solid $focus;
+  }
+  .#{$prefix}--input_container-heading {
+    position: relative;
+    cursor: pointer;
+    padding: $spacing-03 $spacing-05 $spacing-03 $spacing-05;
+    color: $text-02;
+    @include use-carbon-productive-tokens();
```

Comment:
More of a note for ourselves, we should update on the sunset branch to remove this mixin once this is merged in.